pkg/cosign/verify.go

```diff
@@ -84,6 +86,11 @@ type CheckOpts struct {
 
 	// SignatureRef is the reference to the signature file
 	SignatureRef string
+
+	// Identities is an array of Identity (Subject, Issuer) matchers that have
+	// to be met for the signature to ve valid.
+	// Supercedes CertEmail / CertOidcIssuer
```

Comment:
Can we change those to pass a `[]v1alpha1.Identity`?

Reply:
Thought about it, but did not want to change them because they would match other things than email and that would be a change in behaviour. So, I'd rather not do that as part of this PR.